mungegithub/mungers/approvers/approvers_test.go

```diff
@@ -309,13 +310,34 @@ func TestGetCCs(t *testing.T) {
 			// We don't suggest approvers for a and b, only for unapproved c.
 			expectedCCs: []string{"Carol"},
 		},
+		{
+			testName:  "A, B, C; Nothing approved, but assignees can approve",
+			filenames: []string{"a/test.go", "b/test.go", "c/test"},
+			testSeed:  0,
+			// Approvers are valid approvers, but not the one we would suggest
+			currentlyApproved: sets.NewString(),
+			assignees:         []string{"Art", "Ben"},
+			// We suggest assigned people rather than "suggested" people
+			expectedCCs: []string{"Art", "Ben", "Carol"},
+		},
+		{
+			testName:          "A, B, C; Nothing approved, but SOME assignees can approve",
+			filenames:         []string{"a/test.go", "b/test.go", "c/test"},
+			testSeed:          0,
+			currentlyApproved: sets.NewString(),
+			// Assignees are a mix of potential approvers and random people
+			assignees: []string{"Art", "Ben", "John", "Jack"},
+			// We suggest assigned people rather than "suggested" people
+			expectedCCs: []string{"Art", "Ben", "Carol"},
+		},
```

Comment:
That's what the test is doing, I'll add an extra comment to make it clearer but:

> assignees: []string{"Art", "Ben", "John", "Jack"},
> expectedCCs: []string{"Art", "Ben", "Carol"},

But if you look at the test above, completely similar with no-one assigned, you have the following suggested:
> testName:          "A, B, C; Nothing Approved",
> ...
> expectedCCs: []string{"Anne", "Bill", "Carol"},

So we can see that it's suggesting assignees Art and Ben rather than Anne and Bill.